src/screens/lock/LockScreen.tsx

```diff
@@ -131,6 +134,15 @@ export const LockScreen = (props: LockScreenProps): JSX.Element => (
   <>
     <FlagshipBars />
 
-    <LockView {...useLockScreenProps(props.route?.params)} />
+    <TouchableWithoutFeedback
+      onPress={Keyboard.dismiss}
```

Comment:
This dismiss the keyboard, but does it remove the focus on text field too?

Reply:
it does indeed